src/include/platform/CHIPDeviceConfig.h

```diff
@@ -196,7 +196,7 @@
  * A monothonic number identifying the firmware revision running on the device.
  */
 #ifndef CHIP_DEVICE_CONFIG_DEVICE_FIRMWARE_REVISION
-#define CHIP_DEVICE_CONFIG_DEVICE_FIRMWARE_REVISION 1
+#define CHIP_DEVICE_CONFIG_DEVICE_FIRMWARE_REVISION 0
```

Comment:
On similar lines, I did not find anything related to this in ESP32 Kconfig. Will you please investigate?

Reply:
Yeah, this option is not exposed in Kconfig.